### PR TITLE
[ACS-4530] Add ACA libraries upstream PR to monorepo

### PR DESCRIPTION
**Pull request by MichalKinas**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

ACA libraries published to NPM are not updated in monorepo. https://alfresco.atlassian.net/browse/ACS-4530

**What is the new behaviour?**

Upstream PR for monorepo will be created after ACA libraries are updated.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: scripts/travis/update/aca-same-commit-verify.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    program
+        .version('0.1.0')
+        .option('-t, --token [type]', 'token')
+        .option('-h, --head [type]', 'head')
+        .option('-r, --repo [type]', 'repo')
+        .option('-c, --commit [type]', 'commit')
+        .parse(process.argv);
+
+    const { token, head, repo, commit } = program,
+        prCreator = new PrCreator(ORGANISATION, repo, token, commit);
```
Comment (BSekula) [inline scripts/travis/update/aca-same-commit-verify.js:47]: can we have as new declaration? i.e. 

```
const { token, head, repo, commit } = program;
const prCreator = new PrCreator(ORGANISATION, repo, token, commit);
```

I think it is easier to read.

File: scripts/travis/update/pr-creator.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            await this.repoDestination.updatePullRequest(upstreamPrOpen.number, { title, body: `sha:${commit}` });
+            return upstreamPrOpen.number;
+        }
+
+    }
+
+    async fetchContributors(shaFrom, shaTo)  {
+        const mapAuthors = new Map();
+        let upstreamShaFound = true;
+        const listCommits = await this.repoOrigin.listCommits(({sha: shaFrom}))
```
Comment (BSekula) [inline scripts/travis/update/pr-creator.js:43]:
```suggestion
        const listCommits = await this.repoOrigin.listCommits(({sha: shaFrom}));
```

File: scripts/travis/update/pr-creator.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        for (let key of contributors.keys()) {
+            names.push(`@${key}`)
+        }
+        return names.join(', ');
+    }
+
+    async getShaTo(head, base)  {
+        const { data: closedUpstreamPRs }  = await this.repoDestination.listPullRequests({ state: 'closed', head: `${ORGANISATION}:${head}`, base });
+        const latestClosedUpstream = closedUpstreamPRs[0];
+        const shaTo = latestClosedUpstream.body.split(':')[1].trim();
```
Comment (BSekula) [inline scripts/travis/update/pr-creator.js:86]: do we need to check if there is any `latestClosedUpstream`? first time when it is going to be run, there wont be any I think.

File: scripts/travis/update/update-project.sh [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        BRANCH_CREATED=true
+        git checkout -b $BRANCH_TO_CREATE origin/develop
+    fi
+
+    update_dependency "@alfresco/aca-shared"
+    update_dependency "@alfresco/aca-content"
+    update_dependency "@alfresco/aca-about"
+    update_dependency "@alfresco/aca-preview"
+    update_dependency "@alfresco/aca-viewer"
+    update_dependency "@alfresco/aca-folder-rules"
```
Comment (BSekula) [inline scripts/travis/update/update-project.sh:85]: we are missing `@alfresco/aca-settings` and `@alfresco/adf-office-services-ext`

**Review by BSekula — APPROVED** [review]

File: scripts/travis/update/aca-same-commit-verify.js (hunk 1 above)
Comment (MichalKinas) [reply scripts/travis/update/aca-same-commit-verify.js]: Changed

File: scripts/travis/update/pr-creator.js (hunk 3 above)
Comment (MichalKinas) [reply scripts/travis/update/pr-creator.js]: Right, added check for this case

File: scripts/travis/update/update-project.sh (hunk 4 above)
Comment (MichalKinas) [reply scripts/travis/update/update-project.sh:85]: Added

**Pull request merged**